simulation/collectors.py

```diff
@@ -277,6 +277,10 @@ def model_heat_pump_installations_at_current_step(model) -> int:
     return model.heat_pump_installations_at_current_step
 
 
+def model_heat_pump_awareness(model) -> int:
+    return model.heat_pump_awareness
```

Comment:
Is this the right variable? Should you have the `at_current_step` suffix?

Reply:
This is referring to the property `DomesticHeatingABM.heat_pump_awareness` defined in `models.py` which returns `households_heat_pump_aware_at_current_step / household_count`, which is what I want.